tests/phpunit/tests/media.php

```diff
@@ -5440,6 +5440,101 @@ public function test_wp_get_loading_optimization_attributes_image_before_loop_in
 		$this->assertSame( 1, wp_increase_content_media_count( 0 ) );
 	}
 
+	/**
+	 * Tests for pre_wp_get_loading_optimization_attributes filter.
+	 *
+	 * @ticket 58893
+	 */
+	public function test_pre_wp_get_loading_optimization_attributes_filter() {
+		add_filter(
+			'pre_wp_get_loading_optimization_attributes',
+			static function ( $loading_attrs ) {
+				$loading_attrs['fetchpriority'] = 'high';
```

Comment:
```suggestion
				if ( false === $loading_attrs ) {
					// Initialize as an empty array.
					$loading_attrs = array();
				}
				$loading_attrs['fetchpriority'] = 'high';
```

As per the comment by @felixarntz on https://github.com/WordPress/wordpress-develop/pull/5176#discussion_r1330596737, it is necessary to incorporate this adjustment to prevent the occurrence of the `Automatic conversion of false to array is deprecated` error in PHP 8.1 and subsequent versions.

Reply:
Check https://github.com/WordPress/wordpress-develop/actions/runs/6242000649/job/16945177959?pr=5176 for failaur.

Reply:
Thanks @mukeshpanchal27. Either I missed that comment, or it was not clear to me. I think it was the latter, as I believed that Felix was mentioning the tests for the case of providing the false and the empty array as values to that filter.